### PR TITLE
build: Streamline kinetik build and release process

### PR DESCRIPTION
**Pull request by Eknze**
Build and upload kinetik binaries as artefacts so they can be easily re-used throughout the pipeline. For example, in the GitHub release, in a future RPM or Bottle etc.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
+FROM --platform=$BUILDPLATFORM fedora:37 as builder
+WORKDIR /binaries
+COPY --chmod=555 .binaries/ ./
 
 # https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
 ARG TARGETPLATFORM
 
-# The final image uses a 'scratch' image, which requires a statically linked binary.
-ENV RUSTFLAGS='-C target-feature=+crt-static'
+# Install ansible into install folder of the fedora base image
```
Comment (aredgwell) [inline Dockerfile:10]: This does not install ansible..

Comment (Eknze) [reply Dockerfile:10]: Oops!

**Review by aredgwell — APPROVED** [review]
Looks good 👍

**Comment by Eknze** [comment]
https://github.com/Eknze/kinetik-base/actions/runs/4685759382

**Pull request merged**